packages/datetime/package.json

```diff
@@ -39,9 +39,6 @@
     "@blueprintjs/karma-build-scripts": "^0.4.0",
     "@blueprintjs/node-build-scripts": "^0.4.0",
     "@blueprintjs/test-commons": "^0.4.0",
-    "enzyme": "^3.3.0",
```

Comment:
you still need `enzyme` here to adhere to `no-implicit-dependencies` (which we need to turn on at some point)

Reply:
just enzyme, right? no adapter?